### ISSUE TITLE
CustomerFrontend::Navigation###ExternalURLJump###1 impossible to enable

### ISSUE THREAD

**Issue by ghost** [RotherOSS/otobo#1975]
When i try to enable the setting, it tells me that the field NavBar ist mandatory. I am not able to set this setting, if i cannot enable it.
![image](https://user-images.githubusercontent.com/84412466/196057806-72b46eaa-66f0-435b-8726-05ff46bf6650.png)

I tried to Fix and Rebuild the Sysconfig, but that does not solve the issue.

**Comment by bschmalhofer** [comment]
I can confirm that with OTOBO 10.1.x. It looks like the default configuration for CustomerFrontend::Navigation###ExternalURLJump###1 is not valid indeed. This needs to be fixed in _Kernel/Config/Files/XML/Framework.xml_. I'll provide a fix for 10.1.6.

**Comment by bschmalhofer** [comment]
The merged PR looks fine. With a fresh installation using the HEAD of rel-10_1 I could activate `CustomerFrontend::Module###ExternalURLJump` and `CustomerFrontend::Navigation###ExternalURLJump###1` in the system configuration GUI. After that a extra entry with the icon O showed up in the left side menu of the customer interface. Clicking on the O opens a new tab with https://otobo.de/de/. Closing this issue.

Actually, it is not obvious what the setting NavBar does. I suspect that the actual content does not matter, it just has to have a true value. But improving this is a different issue.

I did not re
